scopes/react/ui/apply-providers/apply-providers.tsx

```diff
@@ -1,28 +1,25 @@
 import React, { HTMLAttributes } from 'react';
 import { ErrorBoundary } from 'react-error-boundary';
-import classNames from 'classnames';
 import { Composer } from '@teambit/base-ui.utils.composer';
 import { ErrorFallback } from '@teambit/react.ui.error-fallback';
 
-import { RenderingContext } from '@teambit/preview';
+import type { RenderingContext } from '@teambit/preview';
 import { ReactAspect } from '@teambit/react';
-import styles from './apply-providers.modules.scss';
-
 
 export type ApplyProvidersProps = {
   renderingContext: RenderingContext;
   children: React.ReactNode;
 } & HTMLAttributes<HTMLDivElement>;
 
 /**
- * base template for react component documentation
+ * applies providers from rendering context, and error boundary
  */
 export function ApplyProviders({ renderingContext, children, ...rest }: ApplyProvidersProps) {
 
   const { providers = [] } = renderingContext.get(ReactAspect.id) || {};
 
   return (
-    <div className={classNames(styles.docsMainBlock)} {...rest}>
+    <div {...rest}>
```

Comment:
I think you can remove this div. If someone wants to add properties to ApplyProviders, he can just apply a div above it and get the same results.